docs/reference/launch-configuration.md

```diff
@@ -36,13 +37,23 @@ The file may be called by any name but should have the following structure:
         }
       ]
     }
-  ]
+  ],
+  "streamingQueryInterval": 4,
+  "runtimeFlags": [
+    {
+      "name": "...",
+      "value": "..."
+    }
+  ]   
 }
 ```
 
 | Field | Required/Optional | Description | 
 | :------------- | :------------- | :------- |
+| `template` | Required | Defines the compute resources your deployment needs (i.e., its ‘topology’). See [deployment templates](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#templates) for more details. |
 | `dimensionsInWorldUnits` | Required | Describes how big the world should be. "x" and "z" parameters must both be greater than 0. |
 | `loadBalancing` | Required | The new load balancing configuration as documented in [load balancing with layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancing). |
 | `snapshot` | Optional | Specifies which snapshot file to start the deployment from and how often to take a snapshot while the deployment is running. <br><br> It contains two optional fields: `takeSnapshotIntervalSeconds` (optional) and `startDeploymentFromSnapshotFile` (optional). |
-| `workerFlags` | Optional | Specifies any additional flags to pass to workers. Each element contains two fields: `workerType` (required) and `flags` (required). |
\ No newline at end of file
+| `workerFlags` | Optional | Specifies any additional flags to pass to workers. Each element contains two fields: `workerType` (required) and `flags` (required). |
```

Comment:
```suggestion
| `workerFlags` | Optional | Specifies any additional flags to pass to worker instances. Each element contains two fields: `workerType` (required) and `flags` (required). |
```

Reply:
This is more about a global config that workers can read. I'll rephrase.